src/components/layout/Header.js

```diff
@@ -97,14 +97,18 @@ HeaderSection.defaultProps = {
 }
 
 const H2Header = styled(BaseH2)`
-  padding: 0 !important;
-  margin: 0 !important;
-  font-size: 1.16rem !important;
-  color: ${WHITE};
-  font-weight: normal;
   text-shadow: 1px -1px 17px ${reactBlue(0.4)};
 `
 
+H2Header.defaultProps = {
+  color: 'white',
```

Comment:
same questions re contstant for color

Reply:
the value of the default prop color is "white" per this line https://github.com/reactgraphqlacademy/reactgraphqlacademy/blob/ds-fonts/src/config/styles.js#L122. It means the prop needs to match a key (name of a color) of that object. The variable "WHITE" references the value "#fff" https://github.com/reactgraphqlacademy/reactgraphqlacademy/blob/ds-fonts/src/config/styles.js#L20, so we can't use the const WHITE to match the key in the theme cause "#fff" is not the name of a color. I think this is solved nicely with types, as we do in UM with TypeScript. Without types, I don't see a simple solution for this 🤔